catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -811,6 +814,53 @@ public void setDistanceTolerance(int distanceTolerance) {
     }
   }
 
+  public void setPollInterval(Integer interval) {
+    if (interval == null || interval.equals(pollInterval)) {
+      return;
+    }
+
+    this.pollInterval = interval;
+    LOGGER.debug(
+        "Setting Availability poll task for {} minute(s) on Source {}", pollInterval, getId());
```

Comment:
✏️ 	This wording is confusing. I would consider something like
```suggestion
        "Setting health check on source {} to poll every {} minute(s) ", getId(),  pollInterval);
```